lib/ssl/src/ssl_record.hrl

```diff
@@ -180,4 +180,17 @@
           next_iv  % opaque IV[SecurityParameters.record_iv_length];
          }). 
 
+
+-define('TLS-1.X', {3, _}).
+-define('TLS-1.3', {3, 4}).
+-define('TLS-1.2', {3, 3}).
+-define('TLS-1.1', {3, 2}).
+-define('TLS-1.0', {3, 1}).
+-define('DTLS-1.X', {254, _}).
+-define('DTLS-1.0', {254, 255}).
+-define('DTLS-1.1', {254, 254}).
```

Comment:
This version does not exist! There is only DTLS-1.0 corresponds to TLS-1.1 and DTLS-1.2 corresponds to TLS-1.2